src/Commands/MakeMiddlewareCommand.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Regnerisch\LaravelBeyond\Commands;
+
+use Illuminate\Console\Command;
+use Regnerisch\LaravelBeyond\Resolvers\AppNameSchemaResolver;
+
+class MakeMiddlewareCommand extends Command
+{
+    protected $signature = 'beyond:make:middleware {name?} {--support} {--overwrite}';
+
+    protected $description = 'Make a new middleware';
+
+    public function handle(): void
+    {
+        try {
+            $name = $this->argument('name');
+            $support = $this->option('support');
+            $overwrite = $this->option('overwrite');
+
+            $stub = $support ? 'middleware.support.stub' : 'middleware.stub';
+            $directory = $support ? 'Packages/Laravel/Middleware' : 'Middleware';
```

Comment:
Are we still following Spaties principles by this implementation? Idk honestly

Reply:
Yes. As I remember middlewares are put to `Support` and app specific middlewares are put inside `App`